third_party/compute_library/build_defs.bzl

```diff
@@ -1,6 +1,6 @@
 def if_enable_acl(if_true, if_false = []):
     return select({
-        "@org_tensorflow//third_party/compute_library:build_with_acl": if_true,
```

Comment:
I think we should keep the full path (`@org_tensorflow//...`). Otherwise, other repositories that depend on TF may have build issues (they won't know `@compute_library`). Same for other places.